lexicons/app/bsky/unspecced/getPostThreadV2.json

```diff
@@ -0,0 +1,144 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.getPostThreadV2",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "(NOTE: this endpoint is under development and WILL change without notice. Don't use it until it is moved out of `unspecced` or your application WILL break) Get posts in a thread. It is based in an anchor post at any depth of the tree, and returns posts above it (recursively resolving the parent, without further branching) and below it (recursive replies, with branching). Does not require auth, but additional metadata and filtering will be applied for authed requests.",
+      "parameters": {
+        "type": "params",
+        "required": ["anchor"],
+        "properties": {
+          "anchor": {
+            "type": "string",
+            "format": "at-uri",
+            "description": "Reference (AT-URI) to post record."
+          },
+          "above": {
+            "type": "integer",
+            "description": "How many levels of parent (and grandparent, etc) to include above the anchor.",
+            "default": 80,
+            "minimum": 0,
+            "maximum": 100
+          },
```

Comment:
I'm still a little skeptical we need this parameter at all, or that if we do need it, it could be a boolean to include all ancestors (up to an app-defined limit).

Reply:
May need to revisit this once we do some testing with some prod data